data-plane/tests/src/test/java/dev/knative/eventing/kafka/broker/tests/DataPlaneTest.java

```diff
@@ -317,18 +335,18 @@ private static ConsumerDeployerVerticle setUpDispatcher(final Vertx vertx, final
     return verticle;
   }
 
-  private static ReceiverVerticle setUpReceiver(
-    final Vertx vertx,
-    final VertxTestContext context) throws InterruptedException {
-
-    final Function<Vertx, RequestMapper<String, CloudEvent>> handlerFactory = v -> new RequestMapper<>(
-      vertx,
-      producerConfigs(),
-      new CloudEventRequestToRecordMapper(v),
-      properties -> KafkaProducer.create(v, properties),
-      mock(Counter.class),
-      mock(Counter.class)
-    );
+  private static ReceiverVerticle setUpReceiver(final Vertx vertx, final VertxTestContext context)
```

Comment:
It wrapped the `throws`?! that's bad

Reply:
google-java-format is design to give a way to change style, bad or good, it's consistent, like in Go